### ISSUE TITLE
raster.sample() index is lastUpdated datetime rather than acquired datetime

### ISSUE THREAD

**Issue by scottyhq** [SlideRuleEarth/sliderule#444]
I think this is a shortcoming of the national map API rather than sliderule, but wanted to document that it's not obvious timestamps returned from raster.sample are when the product was 'last updated' rather than when the data was acquired.

```python
region = [  {"lon": -108.34, "lat": 38.89},
            {"lon": -107.76, "lat": 38.90},
            {"lon": -107.78, "lat": 39.26},
            {"lon": -108.36, "lat": 39.25},
            {"lon": -108.34, "lat": 38.89}  ]

geojson = earthdata.tnm(short_name='Digital Elevation Model (DEM) 1 meter', polygon=region)
gdf = raster.sample("usgs3dep-1meter-dem", [[-108.1,39.1]], {"catalog": geojson})
gdf
```
<img width="670" alt="image" src="https://github.com/user-attachments/assets/0896c6d5-3d9b-46b7-af88-3b3369da5196">

This is inconsistent with other APIs where 'time' is observation time of the data. 

The timestamps are coming from here 
https://github.com/SlideRuleEarth/sliderule/blob/20a5b41617fe64def65a7ade7276bea109ee58c7/clients/python/sliderule/earthdata.py#L788

Unfortunately I don't think the national map api keeps track of the nominal acquisition date of a given 1m tile?...

**Comment by scottyhq** [comment]
Yes, it's really unfortunate. I sent an email to USGS TNM support and cc'd you as it would be best if the metadata were present in the API response. 

It is possible to workaround this by getting collect_start and collect_end from the `project` name here https://prd-tnm.s3.amazonaws.com/StagedProducts/Elevation/metadata/WESM.csv 

```python 
import pandas as pd
df = pd.read_csv('https://prd-tnm.s3.amazonaws.com/StagedProducts/Elevation/metadata/WESM.csv')
df[df.project == 'LA_2021GreaterNewOrleans_C22']
df.iloc[0]
```

**Comment by elidwa** [comment]
We are postponing further work on this issue until accurate tile-level acquisition metadata becomes available directly in the raster files. Currently, the 1-meter 3DEP tiles do not provide reliable acquisition dates.

As noted in a response from the TNM Help Desk (forwarded to @scottyhq):

"You can perhaps take solace in the fact that you are not the first one to request this. The challenge that we've had with this is that acquisition dates within 1-meter tiles (10km x 10km) can often be different within the tile or across tiles within the project. In the 'early' days of 3DEP, we had inconsistent metadata and were not able to readily determine tile-level collection dates. Now, this is a required delivery and we are talking about how to provide 'accurate' tile-level acquisition, publication, and revision dates.

I don't have a timeline on when this will be implemented. However, I will pass along this request to our leadership to help prioritize the effort."

Once this feature is implemented and acquisition dates are reliably embedded in the tile metadata, I will revisit the code and update it to extract and use the correct date from each tile.
